src/controller/python/chip/CertificateAuthority.py

```diff
@@ -197,6 +206,17 @@ def maximizeCertChains(self, enabled: bool):
 
         self._maximizeCertChains = enabled
 
+    @certificateValidityPeriod.setter
+    def certificateValidityPeriod(self, validity: int):
```

Comment:
throught: Should have `Sec` suffix or values should be of some timedelta or typed type, so that we do not make seconds/ms or other conversion errors.

Reply:
Yes, you're right. I have updated it to use `certificateValidityPeriodSec` to avoid conversion errors.